models/PromoBlock.php

```diff
@@ -145,6 +145,9 @@ class PromoBlock extends Model
         'date_end',
     ];
 
+    public $visible = [];
+    public $hidden = [];
```

Comment:
@kharanenka I wonder if this would be a problem, because the name `hidden` collide with the column name of lovata_shopaholic_promo_block table. That was why I didn't add it here.

Reply:
Yes you are right, I need to change field name in database to "is_hidden"